src/content/docs/merchants/get-started/multistore.mdx

```diff
@@ -96,22 +103,47 @@ folders:
   /en_ca/products/: /en_ca/products/default`;
 ```
 
-## Store-specific files
-
-The content, placeholders, and analytics files help define the experience in each store view as described here.
-
-### Content files
-
-The content files provide the structure of the pages served to your shoppers. They are located in the store view directories and contain the content for each store view. These files are typically SharePoint `.docx` or Google document files used to generate the store view pages.
-
+#### Config Service
 
-### Placeholders
+If your project uses the Config Service, you must update the `default-config.json` file to add the new store view mapping under the `folders:` section, like this:
 
-The `placeholders.xlsx` file stores reusable tokens for text and UI components. This file is used to define the text that is displayed on your storefront. The `placeholders.json` file is generated from this file and is used to replace the tokens in your content files. Optionally, you can create a `placeholders-overrides.json` file to define text values that supersede those specified in the `en` store view. For more information, see [Placeholders](/resources/placeholders).
+```json
+{
+  "folders": {
+    "/en/products/": "/en/products/default",
+    "/en_ca/products/": "/en_ca/products/default"
+  }
+}
+```
 
-### Analytics
+Follow the instructions in the [Storefront Configuration](/setup/configuration/commerce-configuration/) to publish your updates to the Config Service.
```

Comment:
I see you added this - I will have to update that page to include these steps then.

I think you should instead point people to helix docs. I do not include specifics of how to manage aem config on the commerce-configuration page.

- https://www.aem.live/docs/config-service-setup
- https://www.aem.live/docs/admin.html#schema/SiteConfig
- https://www.aem.live/docs/admin.html#tag/siteConfig/operation/updateConfigSite